Documentation/GeneralConventions/ReadmeFile.rst

```diff
@@ -0,0 +1,95 @@
+
+.. index:: File structure; README.rst, README.rst
+.. _readme-rst:
+.. _about-file:
+
+=======================================
+:file:`README.rst` or :file:`README.md`
+=======================================
+
+Full documentation contains both a README.rst and a Documentation/Index.rst
```

Comment:
I don't like the phrase "full documentation" in this context.

So, this sentence says: Just add a README.rst and a Index.rst file and you have full documentation. That's it.

And why do I need a README.rst file?

Reply:
I also dont get all of this, so I moved it to a separate file so it can be dealt with later. I didnt change the content, see attached commit